### PR TITLE
Dev01235ipv6

### PR DESCRIPTION
**Pull request by camenlx**
01coin code changes:
-allow ipv6 and ipv4 masternodes
-instasend up to 100000 zoc
-enforce mn min protocol version 70210 and then 70211 after 5-July
-enforce enabled to earn mn rewards (sentinel now mandatory) after 5-July
-zuffs instead of duffs
-2019 year bump on some changed files
-version string at wallet, deamon and cli client
-small typos ZOCifying code and comments into ZOC, defaultport 10000 and testport 10002

### CODE REVIEW

File: src/chainparamsseeds.h [hunk 1]
```diff
@@ -9,24 +9,24 @@
...
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2e,0xa2,0x42,0x0a}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x4e,0x6d,0xb2,0xc3}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x8a,0x80,0xa9,0x5e}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x34,0x0b,0x8d,0xe5}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x25,0x3b,0x15,0x3a}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x2e,0x69,0x76,0x0f}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0xb2,0x21,0x7e,0xdd}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x68,0xec,0x17,0x83}, 9999},
-    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x6c,0x3d,0xd1,0x25}, 9999}
+    // {{0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0x00,0xff,0xff,0x80,0x7f,0x6a,0xeb}, 10000},
```
Comment (OwenRay) [inline src/chainparamsseeds.h]: We should add our own seed addresses here

File: src/privatesend.cpp [hunk 2]
```diff
@@ -514,14 +515,14 @@ void CPrivateSend::SyncTransaction(const CTransaction& tx, const CBlockIndex *pi
...
-    if(fLiteMode) return; // disable all Dash specific functionality
+    if(fLiteMode) return; // disable all ZOC specific functionality
 
     static bool fOneThread;
     if(fOneThread) return;
     fOneThread = true;
 
     // Make this thread recognisable as the PrivateSend thread
-    RenameThread("dash-ps");
+    RenameThread("zeroone-ps");
```
Comment (OwenRay) [inline src/privatesend.cpp:525]: be carefull changing thread names, did you check if this threadname is referenced anywhere else?

**Review by OwenRay — CHANGES_REQUESTED** [review]
Looks great:)
I made two comments that need your attention

File: src/privatesend.cpp (hunk 2 above)
Comment (camenlx) [reply src/privatesend.cpp:525]: yes, I did a "Find in files" with Notepad++ :
// string match at init.cpp#L1942
    // ********************************************************* Step 11d: start zeroone-ps-<smth> threads
// string match at privatesend-client.cpp#L1463
    RenameThread("zeroone-ps-client");
// string match at privatesend-server.cpp#L904
    RenameThread("zeroone-ps-server");
// string match at privatesend.cpp#L525
    RenameThread("zeroone-ps");

File: src/chainparamsseeds.h (hunk 1 above)
Comment (camenlx) [reply src/chainparamsseeds.h]: done, see bellow commit for this: https://github.com/zocteam/zeroonecoin/pull/33/commits/f4abeeef80021d46e6803860438333c66d9dd60a

**Comment by camenlx** [comment]
reviewer needs to go to at least to 1 module and select accept on the changes to be able to merge into master sources for finaly be able to build ipv6 node sw

**Review by OwenRay — APPROVED** [review]

**Pull request merged**